lib/node_modules/@stdlib/stats/incr/nanmmeanvar/lib/main.js

```diff
@@ -0,0 +1,84 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2018 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MODULES //
+var isnan = require( '@stdlib/math/base/assert/is-nan' );
+var incrmmeanvar = require( '@stdlib/stats/incr/mmeanvar' );
+var isArrayLike = require( '@stdlib/assert/is-array-like' );
+
+
+// MAIN //
+
+/**
+* Returns an accumulator function which incrementally computes a moving arithmetic mean and unbiased sample variance, skipping NaN values.
+*
+* @param {(ArrayLike|PositiveInteger)} out - output array or window size
+* @param {PositiveInteger} [W] - window size
+* @throws {TypeError} must provide a positive integer number for window size
+* @returns {Function} accumulator function
+*
+* @example
+* var acc = incrnanmmeanvar( 3 );
+* acc( 2.0 );  // => [ 2.0, 0.0 ]
+* acc( NaN );  // => [ 2.0, 0.0 ] (NaN skipped)
+* acc( 4.0 );  // => [ 3.0, 2.0 ]
+*
+* @example
+* var out = [ 0.0, 0.0 ];
+* var acc = incrnanmmeanvar( out, 3 );
+* acc( 2.0 );
+* console.log( out ); // => [ 2.0, 0.0 ]
+*/
+function incrnanmmeanvar( out, W ) {
+	var mmeanvar;
+
+	// Detect signature: (W) or (out, W)
+	if ( isArrayLike( out ) ) {
+		mmeanvar = incrmmeanvar( out, W );
+	} else {
+		mmeanvar = incrmmeanvar( out );
+	}
+
+	return accumulator;
+
+	/**
+	* Accumulator function.
+	*
+	* @param {number} [x] - input value
+	* @returns {(Array|null)} current mean and variance
+	*/
+	function accumulator( x ) {
+		// If no arguments → return current state:
+		if ( arguments.length === 0 ) {
+			return mmeanvar();
+		}
+		// Skip NaNs (do not update):
+		if ( isnan( x ) ) {
+			return mmeanvar();
+		}
+		// Update with valid number:
+		return mmeanvar( x );
+	}
+}
+
+
+// EXPORTS //
+
+module.exports = incrnanmmeanvar;
```

Comment:
```suggestion
module.exports = incrnanmmeanvar;
```

This file is missing a trailing newline.